revops/index.html

```diff
@@ -56,15 +61,30 @@ <h2 class="section-heading">Our Expertise</h2>
       </div>
     </div>
     <div class="collection-list-wrapper w-dyn-list">
-      <div class="collection-list w-dyn-items" id="services"></div>
+      <div class="collection-list w-dyn-items">
+        {% for s in page.services %} {% for service in site.services %} {% if s
+        == service.slug %}
```

Comment:
Cannae read this

Reply:
Addressed in https://github.com/derangeddk/deranged.dk/pull/100/commits/4c2a1982ecdbe8aa65d3400b8e8b9fd0be703bd8